Neos.ContentRepositoryRegistry/Classes/Command/ContentStreamCommandController.php

```diff
@@ -27,10 +27,13 @@ class ContentStreamCommandController extends CommandController
      * By default, only content streams in STATE_NO_LONGER_IN_USE and STATE_REBASE_ERROR will be removed.
      * If you also call with "--removeTemporary", will delete ALL content streams which are currently not assigned
      * to a workspace (f.e. dangling ones in FORKED or CREATED.).
+     *
+     * @param string $contentRepository The name of the content repository. (Default: 'default')
+     * @param boolean $removeTemporary
```

Comment:
Can we add a short explanation for this argument (that will show up when using `./flow help`)